gflownet/gflownet.py

```diff
@@ -57,8 +57,16 @@ def __init__(
         self.mask_source = self._tbool([self.env.get_mask_invalid_actions_forward()])
         # Continuous environments
         self.continuous = hasattr(self.env, "continuous") and self.env.continuous
+        if self.continuous and optimizer.loss in ["flowmatch", "flowmatching"]:
+            print(
+                """
+            Flow matching loss is not available for continuous environments.
+            Trajectory balance will be used instead
+            """
+            )
+            optimizer.loss = "tb"
         # Loss
-        if optimizer.loss in ["flowmatch"]:
+        if optimizer.loss in ["flowmatch", "flowmatching"]:
```

Comment:
I'm sorry, but what is `flowmatching`? I mean why would it be an option in the config when we already have `flowmatch`?

Reply:
of this is just to allow for both names ("flowmatch" or "flowmatching"), like for the trajectory balance loss we allow both "trajectorybalance" and "tb". Maybe we should add "fm" to the flowmatch loss too :P

Reply:
lol okay